curriculum/challenges/english/03-front-end-libraries/react/write-a-react-component-from-scratch.english.md

```diff
@@ -29,7 +29,8 @@ tests:
     testString: assert((function() { const mockedComponent = Enzyme.mount(React.createElement(MyComponent)); return mockedComponent.find('h1').text() === 'My First React Component!'; })());
   - text: <code>MyComponent</code> should render to the DOM.
     testString: assert(document.getElementById('challenge-node').childNodes.length === 1);
-
+  - text: <code>MyComponent</code> should have a constructor with <code>props</code> calling <code>super</code>.
```

Comment:
```suggestion
  - text: <code>MyComponent</code> should have a constructor calling <code>super</code> with <code>props</code>.
```

I recommend changing the verbiage here, as "props calling super" doesn't seem clear or accurate. 🙂 The suggested verbiage here comes from [this comment](https://github.com/freeCodeCamp/freeCodeCamp/issues/39301#issuecomment-663088086).